Fishing Game/ViewControllers/FishingScreenViewController.swift

```diff
@@ -84,6 +123,20 @@ class FishingScreenViewController: UIViewController {
         
         moveReelToDefaultNoFishPosition()
     }
+    
+//    override func viewWillLayoutSubviews() {
```

Comment:
Let's not leave commented code, let's go a head and delete it if we don't need it